kuma/static/styles/components/payments/_page.scss

```diff
@@ -26,15 +34,15 @@
             @extend %highlight;
             @include set-font-size($h2-font-size);
             margin-bottom: 20px;
+
+            &:only-child {
+                margin-bottom: 0;
+            }
         }
 
         p {
             margin-bottom: 0;
         }
-
-        @media #{$mq-mobile-and-down} {
```

Comment:
We had a few media queries that were `mobile-and-down` and `desktop-and-up`, and I thought it would be more consistent and clear.to make it mobile first and then add media queries for the larger views. The rearranging in this file is a result of that.